### PR TITLE
Update Roles, Buttons, and Naming

### PR DESCRIPTION
**Pull request by wcrum**
1. Made `src/components/buttons/RedButton.svelte` actually a Red button, moved refernces of RedButton (which were white) to `src/components/buttons/WhiteButton.svelte`
2. Updated Roles.
    - Updated md files to correct content.
    - Added frontmatter to support image wide for banner images.
3. Added background matching support for rounded edges.
    - When the previous component to the footer does not match the background, it creates a small clip of the wrong color. Adding an additional div with specified background fixes this.
4. Adds images to navigation.

### CODE REVIEW

File: src/components/cards/CohortGroup.svelte [hunk 1]
```diff
@@ -1,34 +1,51 @@
...
+      <CohortCard />
+      <CohortCard />
     </div>
   </div>
   <!-- Right Column -->
-  <div class="w-1/2 pt-5">
-    <!-- Grid of Cohorts -->
-    <div class="grid grid-cols-2">
-      {#each entries as cohortEntry}
+  <!--
```
Comment (dfullaway) [inline src/components/cards/CohortGroup.svelte]: If we don't need this anymore, I suggest deleting vice commenting out.

File: src/components/cards/CohortGroup.svelte [hunk 2]
```diff
@@ -1,34 +1,51 @@
...
+          class="block text-2xl font-semibold md:text-3xl lg:text-4xl text-black"
+        >
+          JOIN THE COHORT THAT'S RIGHT FOR YOU.
+        </h1>
+      </div>
+    </div>
+  </div>
+  <div class="flex py-20">
+    <div class="flex flex-col lg:flex-row w-full items-center justify-center gap-16">
+      <CohortCard />
```
Comment (dfullaway) [inline src/components/cards/CohortGroup.svelte]: Instead of putting in two blank cohort cards, we should put in all the cohorts like was done in the commented out section. This might need to be done as part of a different PR, as it will require updating the roles pages to actually show anything here.

File: src/components/cards/CohortCard.svelte
Comment (dfullaway) [inline src/components/cards/CohortCard.svelte:1]: You've updated the formatting, but just have filler data, instead of using the exports. Was this just to show data until the other parts are fixed?

**Review by dfullaway — COMMENTED** [review]
Good work. I'm good with merging, assuming we break out the fixing the cohort stuff to pull in the cohort collection and show the data. We will probably also need to ask the question if we always display two cohorts, a maximum of two cohorts (less if there are less on deck), or just however many cohorts have been defined. May also need to filter out any past cohorts.

File: src/components/cards/CohortCard.svelte
Comment (wcrum) [reply src/components/cards/CohortCard.svelte:1]: Yes - this was just filler, for the cohort cards.

Comment (wcrum) [reply src/components/cards/CohortCard.svelte:1]: Fixed in latest commit.

File: src/components/cards/CohortGroup.svelte (hunk 2 above)
Comment (wcrum) [reply src/components/cards/CohortGroup.svelte]: Fixed in latest commit.

**Review by dfullaway — APPROVED** [review]
Looks good!
